src/components/PsAccounts.vue

```diff
@@ -1,35 +1,42 @@
 <template>
   <div>
-    <AccountNotInstalled
-      v-if="!context.psAccountsIsInstalled"
-      :account-is-installed="context.psAccountsIsInstalled"
-    />
+    <template v-if="validationErrors && validationErrors.length">
+      <b-alert variant="danger" show>
+        <p>&lt;PsAccounts&gt; integration: Given context is invalid: {{validationErrors.join(';')}}</p>
+      </b-alert>
+    </template>
     <template v-else>
-      <AccountNotEnabled
-        v-if="!context.psAccountIsEnabled"
-        :account-is-enabled="context.psAccountIsEnabled"
+      <AccountNotInstalled
+              v-if="!validatedContext.psAccountsIsInstalled"
```

Comment:
Indent issue ? 4 spaces instead of 2 ? Same problem in multiple line on this file.